core/src/main/java/net/sourceforge/jnlp/tools/JarCertVerifier.java

```diff
@@ -426,6 +447,9 @@ private VerifyResult verifySigners(final Map<CertPath, Integer> jarSignCount) {
     }
 
     private ZonedDateTime zonedDateTime(final Date date) {
+        if (date == null) {
```

Comment:
this is not what I would assume for such method... maybe rename it

Reply:
that it returns now if the param is null ;) Would not expect that. That this is not mandatory. Was just a comment

Reply:
I do not expect the date to be ever null.
But I am not 100% sure because the date comes from the X509 certificates and I do not know if there is a corner case where it would return null

Reply:
let's stay with the current implementation